### ISSUE TITLE
Can't run the demo

### ISSUE THREAD

**Issue by 0xori** [bobbybee/CommodityTracking#23]
Hi, I'm getting this error when running ./demo

`libdc1394 error: Failed to initialize libdc1394
OpenCV Error: Assertion failed (ssize.area() > 0) in resize, file /build/buildd/opencv-2.4.8+dfsg1/modules/imgproc/src/imgwarp.cpp, line 1824
terminate called after throwing an instance of 'cv::Exception'
  what():  /build/buildd/opencv-2.4.8+dfsg1/modules/imgproc/src/imgwarp.cpp:1824: error: (-215) ssize.area() > 0 in function resize
`

any idea?

**Comment by bobbybee** [comment]
No; however, I'm currently in the process of rewriting CT to work significantly better -- updates to follow! :smile:
